zenodb_test.go

```diff
@@ -365,12 +365,20 @@ view_a:
 		}
 		return true, nil
 	})
-	testAggregateQuery(t, db, now, epoch, resolution, asOf, until, scalingFactor, modifyTable)
+
+	testAggregateQuery(t, db, now, epoch, resolution, asOf, until, scalingFactor, false, modifyTable)
+	testAggregateQuery(t, db, now, epoch, resolution, asOf, until, scalingFactor, true, modifyTable)
```

Comment:
This is in our chain test, which makes sure that the stride stuff is handled correctly end-to-end. It's not a very comprehensive test of logic, just basically making sure the plumbing is in place.

Our chain test runs in three modes:

- No clustering
- Clustering with pushdown disabled
- Clustering with pushdown enabled

That hits the three main different flavors of query plan.